openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -186,12 +186,20 @@ def update_account_settings(requesting_user, update, username=None):
     additional_fields = {"sch_org", "phone", "organization_type"}.intersection(set(update.keys()))
     if additional_fields:
         instance = AdditionalRegistrationFields.objects.filter(user=existing_user).first()
+        data = {}
+        AdditionalRegistrationFields.validate_data(update, additional_fields, data, field_errors)
```

Comment:
data will always be empty?

Reply:
@tasawernawaz data dict will be having validated data after the execution of AdditionalRegistrationFields.validate_data(update, additional_fields, data, field_errors) method.